Completed tasks

CallumCPP opened the pull request:
Merge dev branch with master. Open for reviewers

CallumCPP committed:
Completed tasks

CallumCPP committed:
Completed tasks

FNASmileys commented on CMP1903_A1_2324/Game.cs:
```diff
@@ -16,6 +16,65 @@ internal class Game
+
+        private bool Condition()
+        {
+            if (testingIterations == 0)
```
braces should be used even when optional

FNASmileys commented on CMP1903_A1_2324/Game.cs:
comments needed

FNASmileys commented on CMP1903_A1_2324/Die.cs:
comments needed

FNASmileys commented on CMP1903_A1_2324/Program.cs:
comments needed

FNASmileys commented on CMP1903_A1_2324/Testing.cs:
comments needed

FNASmileys commented on CMP1903_A1_2324/Die.cs:
```diff
@@ -14,10 +14,14 @@ internal class Die
 
         //Property
-
+        public int value = 0;
```
"Value" not "value" as public variables must be in PascalCase

FNASmileys commented on CMP1903_A1_2324/Game.cs:
```diff
@@ -16,6 +16,65 @@ internal class Game
          */
 
         //Methods
+        public List<Die> dice = new List<Die>();
```
public variables must be in PascalCase

FNASmileys commented on CMP1903_A1_2324/Game.cs:
```diff
@@ -16,6 +16,65 @@ internal class Game
 
         //Methods
+        public List<Die> dice = new List<Die>();
+        public int total { get; private set; }
```
public variables must be in PascalCase >:(

mgoodwin-dev commented on CMP1903_A1_2324/Die.cs:
Code meets functionality requirements
public Variables must be PascalCase, private variables must be _camelCase
Good readability but needs comments to show what the code does

mgoodwin-dev commented on CMP1903_A1_2324/Game.cs:
Code meets functionality requirements
public Variables must be PascalCase, private variables must be _camelCase, braces should be used when optional
Good readability but needs comments to explain the code

mgoodwin-dev commented on CMP1903_A1_2324/Program.cs:
Functionality works as intended although the code will say testing completed whether the user decides to test or not
follows the style guide
Comments are needed for better readability

mgoodwin-dev commented on CMP1903_A1_2324/Testing.cs:
Functionality works as intended
follows the style guide
Comments are needed for better readability
Some test cases are checked for but could be more thorough testing to check all cases like the amount of dice rolled is correct

mgoodwin-dev reviewed (commented):
The functionality of the code is met throughout

The style guide is not met throughout, usually when it comes to naming conventions 

Comments are needed throughout the whole project to get better readability aswell as following the style guide fully

Test cases test for the majority of errors and test the die scripts enough times to know it works correctly

FNASmileys committed:
Reformat code to fit google C# style guidelines.
Add comments to all members except in Program.cs
Remove public total variable from game, Play() now returns result.

CallumCPP committed:
Reformat code to fit google C# style guidelines.
Add comments to all members except in Program.cs
Remove public total variable from game, Play() now returns result.

CallumCPP committed:
Merge branch 'dev' of https://github.com/CallumCPP/CMP1903_A1_2324 into dev

CallumCPP committed:
Missed a '.' in Testing.cs

CallumCPP committed:
Merge branch 'dev' of https://github.com/CallumCPP/CMP1903_A1_2324 into dev

CallumCPP committed:
Create README.md
